src/lib/openapi/spec/upsert-segment-schema.ts

```diff
@@ -16,14 +16,30 @@ export const upsertSegmentSchema = {
         project: {
             type: 'string',
             nullable: true,
+            description:
+                'Project from where this segment will be accessible. If none is defined the segment will be global (i.e. accessible from any project).',
         },
         constraints: {
             type: 'array',
+            description:
+                'List of constraints that determine which users will be part of the segment',
             items: {
                 $ref: '#/components/schemas/constraintSchema',
             },
         },
     },
+    example: {
+        name: 'segment name',
+        description: 'segment description',
+        project: 'optional project name',
```

Comment:
I believe in most cases it's the same, but it could still be useful to be explicit.
```suggestion
        project: 'optional project id',
```